packages/map/src/components/Legend/components/index.scss

```diff
@@ -189,7 +189,7 @@
           justify-items: center;
           line-break: loose;
           align-items: center;
-          width: auto;
+          width: auto !important;
```

Comment:
Avoid using !important keyword in favor for a more specific selector.

Reply:
Fixed.